### PR TITLE
Add names from the chat to haskell annotator, update annotated images

### PR DESCRIPTION
**Pull request by adept**

### CODE REVIEW

**Comment by mx00s** [comment]
`s` - Message 18
`c` - Message 19
`b` - Message 20

**Comment by mx00s** [comment]
These are coherent with the generic symbol numbers.

See
* 15da603c
* 9fa7836b
* af7dd148d

**Comment by adept** [comment]
Also includes #47

**Comment by divanorama** [comment]
#47 should invalidate/update some of decoded messages, didn't manage to compile annotate.hs locally yet to verify which ones are affected

File: source/annotate.hs [hunk 1]
```diff
@@ -310,18 +312,35 @@ symRepr (SymOperator val) = (text, "yellow")
...
           , (365, "add")
           -- comparisons
           , (416, "lt")
           , (448, "eq")
+            -- combinators
+          -- B,C,K,W-system (https://en.wikipedia.org/wiki/B,_C,_K,_W_system)
+          , (6, "c")
+          , (5, "b")
+          , (7, "s")
+          , (1, "i")
```
Comment (mx00s) [inline source/annotate.hs:433]: There may be a bug related to `i`. In the original dump (2c4223b) the generic decoding of the symbol was `x0`. Should it have been `:1`?

Comment (mx00s) [reply source/annotate.hs:433]: Same with `isnil`. It's defined using `x0`, so I don't know how we'll recognize it in the wild. How did you connect it to `:15`?

Comment (mx00s) [reply source/annotate.hs:433]: @adept, are the glyphs for `i` (m24) and `isnil` (m29) entirely new? They're not the same as each other.

`i` is a 2x2 filled-in box and `isnil` is a 3x3 filled-in box.

Based on m8, I think the smallest dimensions possible for a variable is 4x4.

Comment (mx00s) [reply source/annotate.hs:433]: Ah, I see. This is fixed in #47.

**Comment by mx00s** [comment]
I have a PR incoming for `i` and `isnil` incorrectly getting interpreted as operators; they're entirely new symbols.

*EDIT:* Oh, I guess they do fit the pattern for operators. Nevermind.

**Pull request closed without merging**